BUILD.md

```diff
@@ -64,14 +64,18 @@ spack install dla-future@master <variants>
 - [umpire](https://github.com/LLNL/Umpire)
 - [blaspp](https://bitbucket.org/icl/blaspp/src/default/)
 - [lapackpp](https://bitbucket.org/icl/lapackpp/src/default/)
-- Intel MKL or other LAPACK implementation
+- BLAS/LAPACK implementations (e.g. Intel MKL, OpenBLAS, Nvidia nvpl)
```

Comment:
```suggestion
- BLAS/LAPACK implementations (e.g. Intel MKL, OpenBLAS, NVPL)
```
? I think NVPL is "NVIDIA Performance Libraries", so probably "NVIDIA NVPL" is redundant?